src/Fozzie.java

```diff
@@ -2,10 +2,28 @@
 
 import fozzie.fuzzer.CommandFactory;
 
+import java.io.*;
+import java.net.*;
+
 public class Fozzie {
 
-  public static void main(String[] args) {
-    for (int i = 0; i < 20; i++)
-      System.out.println(CommandFactory.makeRandomCommand(i));
+  public static void main(String[] args) throws Exception {
+	  
+  	DatagramSocket clientSocket = new DatagramSocket();
+  	InetAddress IPAddress = InetAddress.getByName("192.168.1.1");
+  	
+    for (int i = 0; i < 1; i++){
+    	//String commandString = CommandFactory.makeRandomCommand(i).toString();
```

Comment:
why is this commented out?  Also, I think the CommandFactory API changed, so I think this is going to need to be switched over